src/render/tutorials/getting-started-with-docker/2_0.html.md

```diff
@@ -155,72 +199,216 @@ Add in `routingOperations` array following definition:
   }
 ```
 
+Here we provide not one, but three different handlers! Some of them even have some additional configurations defined explicitly or in another file. You'll learn about `fragments` and their power and functionalities in a different tutorial.
 
-### Task configuration
-As you probably noticed the `Knot.x` snippet you defined in the `books.html` template file pointed to the `bookslist` task. Let's define it.
+Notice two additional imports: `${config.server.handlers.common.request}` and `${config.server.handlers.common.response}`. Here we use the handlers defined in [Knot.x source code](https://github.com/Knotx/knotx-server-http/blob/master/conf/server.conf), which simplifies handling of basic requests without repeating the configuration over and over again for similar operations.
+
+Now that, we have a feeling of it, let's connect our `hello-world-operation` with some handler.
+
+Simply add new routing operation to the array:
 
-Create `knotx/conf/routes/handlers/fragmentsHandler.conf` and add following code:
- 
 ```hocon
-tasks {
-  bookslist {
-    action = books
-    onTransitions {
-      _success {
-        action = te-hbs
+  {
+    operationId = hello-world-operation
+    handlers = [
+      {
+        name = hellohandler
       }
-    }
+    ]
   }
+```
+
+## Creating new handler
+
+Almost there!
+
+Now we'll have to provide a handler to handle our request. Let's put it in a separate module.
+
+Create a new module under the `modules` folder. The structure should look like this:
+
+```
+modules
+└─ hellohandler
```

Comment:
I'd change the name of this module. It suggests that you're supposed to create a seperate module for every single handler

Reply:
@Voycawojka what do you mean?

Do you think that we should provide some `tutorialModule` where we put this `helloWorldHandler` ?

Reply:
renamed to `hello-module`